### ISSUE TITLE
accessibility contrast problem

### ISSUE THREAD

**Issue by hikeislife** [GoogleChrome/lighthouse#10441]
**Summary**

is there a label or html attribute I could use to indicate that element has intentionally lower contrast ratio, as it contains information that is supposed to be hard to read? 

this includes examples of how not to design a page and post meta data (date it was posted on and author), which I'm sure people with bad eyes, like me, love to read, except I hate when I read those on accident, so I prefer them to be of lower visibility. not that they are hard to read, font is huge, and I made a site so that I can read it, and I keep half of the internet zoomed in, but just "harder" to read 

it doesn't make sense to force me to build a site unusable to visually impaired people so I could pass accessibility audit. clearly the way readable contrast is calculated doesn't take human eyes into account.

**Comment by patrickhulce** [comment]
Thanks for filing @hikeislife! I'm inclined to agree there can be some situations where a developer needs to intentionally violate the color contrast rule.

The results of this audit come directly from [axe](https://github.com/dequelabs/axe-core), so maybe you could take this issue up there and see what comes out of it? If not, then this functionality comes down to augmenting Lighthouse's audit options to come up with our own whitelist functionality.

**Comment by exterkamp** [comment]
+1 to @patrickhulce this is an axe problem primarily. It should be taken up with them first.
